### PR TITLE
Apdoslogconf will warning case format is not splunk

### PR DESCRIPTION
**Pull request by pasmant**
Configuration for security log only support filter field now.
To make backward compatibility the field in apdoslogconfs CRD doesnt remove.
The default values are no more needed in the CRD for format and max-message-size.
Content field values will be ignore, and warning will generate when creating new apdoslogconfs with content field.

### CODE REVIEW

**Review by jjngx — APPROVED** [review]

File: pkg/apis/dos/validation/dos.go [hunk 1]
```diff
@@ -90,15 +89,28 @@ func validateResourceReference(ref string) error {
 	return nil
 }
 
+// checkAppProtectDosLogConfContentField check conetent field doesnt appear in dos log
+func checkAppProtectDosLogConfContentField(obj *unstructured.Unstructured) string {
+	_, found, err := unstructured.NestedMap(obj.Object, "spec", "content")
+	if err == nil && found {
+		unstructured.RemoveNestedField(obj.Object, "spec", "content")
+		msg := "Content field doesn't supported, use splunk format."
```
Comment (lucacome) [inline pkg/apis/dos/validation/dos.go:97]:
```suggestion
		msg := "the content field is not supported, defaulting to splunk format."
```

**Review by lucacome — CHANGES_REQUESTED** [review]

File: pkg/apis/dos/validation/dos.go (hunk 1 above)
Comment (pasmant) [reply pkg/apis/dos/validation/dos.go]: Change warning

File: pkg/apis/dos/validation/dos.go [hunk 2]
```diff
@@ -90,15 +89,28 @@ func validateResourceReference(ref string) error {
 	return nil
 }
 
+// checkAppProtectDosLogConfContentField check conetent field doesnt appear in dos log
+func checkAppProtectDosLogConfContentField(obj *unstructured.Unstructured) string {
+	_, found, err := unstructured.NestedMap(obj.Object, "spec", "content")
+	if err == nil && found {
+		unstructured.RemoveNestedField(obj.Object, "spec", "content")
+		msg := "the content field is not supported, defaulting to splunk format."
```
Comment (Jcahilltorre) [inline pkg/apis/dos/validation/dos.go:97]:
```suggestion
		msg := "the Content field is not supported, defaulting to splunk format."
```

**Review by Jcahilltorre — APPROVED** [review]

File: internal/k8s/appprotectdos/app_protect_dos_configuration_test.go [hunk 3]
```diff
@@ -86,10 +85,26 @@ func TestCreateAppProtectDosLogConfEx(t *testing.T) {
...
+						"content": map[string]interface{}{
+							"format": "splunk",
+						},
+						"filter": map[string]interface{}{},
 					},
 				},
 			},
+			expectedLogConfEx: &DosLogConfEx{
+				IsValid:  true,
+				ErrorMsg: "the content field is not supported, defaulting to splunk format.",
```
Comment (lucacome) [inline internal/k8s/appprotectdos/app_protect_dos_configuration_test.go:97]:
```suggestion
				ErrorMsg: "the Content field is not supported, defaulting to splunk format.",
```

**Review by lucacome — APPROVED** [review]

**Pull request merged**